src/scss/_base.scss

```diff
@@ -2,4 +2,67 @@
 @import 'fonts';
 
 @import 'media-background';
-@import 'dashboard';
\ No newline at end of file
+
+//---------------> Static CSS
+/* General CSS */
+  .custom-shadow-box {
+    box-shadow: rgba(99, 99, 99, 0.2) 0px 2px 8px 0px;
+  }
+  .custom-row-sm-cards{
+    height: 180px;
+  }
+  .custom-sm-card {
+    width: 22%;
+    padding: 10px;
+  }
+
+/* Mobile extra small */
+  @media (max-width: 374.98px) {
+    .custom-t-cell {
+        padding:20px 0!important;
+        text-align: center !important;
+    }
+  }
+
+/* Mobile */
+  @media (max-width: 575.98px) {
+    .custom-row-sm-cards{
+        margin-bottom: 10px;
+    }
+    .custom-sm-card{
+    width: 45% !important;
+    }
+    .custom-t-cell {
+        padding: 10px;
+        text-align: center !important;
+    }
+    .lv-cell-p{
+        padding: 20px 0 !important;
+        font-size: 0.5rem !important;
+    }
+    .lv-mob-table {
+        padding: 20px 10px !important;
+        font-size: 0.5rem !important;
+    }
+  }
+
+/* Small devices (landscape phones, less than 768px) */
+  @media (max-width: 768.98px) {
+    .custom-row-sm-cards{
+        height: 340px;
+    }
+    .custom-sm-card{
+        width: 48%;
+        }
+    .custom-sm-card span {
+      font-size: 1rem !important;
+    }
+    .custom-sm-card i {
+      font-size: 2rem !important;
+    }
+  }
+/* Medium devices (tablets, less than 992px) */
+// @media (max-width: 991.98px) {  }
+
+/* Large devices (desktops, less than 1200px) */
+// @media (max-width: 1199.98px) {  }
```

Comment:
All the sass files get compiled into a single style sheet, so there is no need to include these styles in the `_base.scss` file. You can move them into another file, based on function, like `_tables.scss` and `_cards.scss` and then include them here using `@import 'tables';` and `@import 'cards'`.